python-checks/src/main/java/org/sonar/python/checks/FlaskHardCodedSecret.java

```diff
@@ -162,16 +160,16 @@ private static boolean isSensitiveProperty(Expression expression) {
         .map(ExpressionList::expressions)
         .filter(list -> list.size() == 1)
         .map(list -> list.get(0))
-        .map(FlaskHardCodedSecretCheck::getAssignedValue)
+        .map(FlaskHardCodedSecret::getAssignedValue)
         .flatMap(TreeUtils.toOptionalInstanceOfMapper(StringLiteral.class))
         .map(StringLiteral::trimmedQuotesValue)
-        .filter(SECRET_KEY_KEYWORD::equals)
+        .filter(key -> this.getSecretKeyKeyword().equals(key))
         .isPresent();
     } else if (expression.is(Tree.Kind.QUALIFIED_EXPR)) {
```

Comment:
move check of qualified expression to exact implementation from the abstract class since it is needed just for particular rule

Reply:
This is a very good catch, thank you. I made the change in bd8ee36efa8204a5c0bd4c0895860bd5494092f0.